docs/graphics-math/sieve-eratosthenes.md

```diff
@@ -0,0 +1,138 @@
+# Sieve of Eratosthenes
+
+The Sieve of Eratosthenes is a iterative method of discovering primes in the sequence of natural numbers (positive integers starting with `1`). How it works is that you take each number, starting with `2` since `1` is always a factor of everything, and use it as a factor to eliminate all the numbers that are multiples of it. So, for the natural numbers...
+
+```
+2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15...
+```
+
+Starting with `2`, every sucessive **second** number is eliminated leaving:
+
+```
+2, 3, _, 5, _, 7, _, 9, __, 11, __, 13, __, 15...
+```
+
+Next, starting with `3`, every **third** number is eliminated leaving:
+
+```
+2, 3, _, 5, _, 7, _, _, __, 11, __, 13, __, __...
+```
+
+This continues on until you've elimiated all multiples of each number and then reach the limit of the numbers your testing, like `15` in the case of this example. The numbers that remain are the prime numbers frome the sequence of numbers you scanned.
+
+Once the very first pass using `2` is complete, you don't need to test any factors that are ``even`` numbers because they're already eliminated.
+
+## Seive game
+
+```typescript
+enum SieveSteps {
+    list,
+    scan,
+    collect,
+    idle
+}
+
+let spriteList: Sprite[] = null
+let boxSprite: Sprite = null
+let boxX = 0
+let boxY = 0
+let j = 0
+let factor = 2
+let step = SieveSteps.list
+
+game.splash("Sieve of Eratosthenes")
+
+forever(function () {
```

Comment:
Use game.onUpdate

Reply:
Ok, I'll have to re-sync my pauses with a counter or something.

Reply:
Mainly just curiosity, but why not forever here? It is still tied in to run only within the game loop, don't see the downside right away